qp_woltka/woltka.py

```diff
@@ -293,7 +295,16 @@ def woltka(qclient, job_id, parameters, out_dir):
 
     for rank in ['phylum', 'genus', 'species']:
         fp = f'{out_dir}/{rank}.biom'
+
         if exists(fp):
+            # making sure that the tables have taxonomy
+            bt = load_table(fp)
+            metadata = {x: {'taxonomy': x.split(';')}
+                        for x in bt.ids(axis='observation')}
+            bt.add_metadata(metadata, axis='observation')
+            with biom_open(fp, 'w') as f:
+                bt.to_hdf5(f, "shogun")
```

Comment:
```suggestion
                bt.to_hdf5(f, "woltka")
```

I know that string is overall ignored, but should it be woltka instead?